### ISSUE TITLE
Twine2.download.recipe - URLTextSearcher: Error: No match found

### ISSUE THREAD

**Issue by blackthroat** [autopkg/denmoff-recipes#3]
```
URLTextSearcher
{'Input': {'re_pattern': u'https://bitbucket.org/klembot/twinejs/downloads/twine_.*?_osx.zip',
           'url': u'http://twinery.org'}}
URLTextSearcher: No value supplied for CURL_PATH, setting default value of: /usr/bin/curl
URLTextSearcher: No value supplied for result_output_var_name, setting default value of: match
No match found on URL: http://twinery.org
Failed.
```
https://github.com/autopkg/denmoff-recipes/blob/ca4a4eae00ead1ce4fcea9a7d8fbe59f7a2a7fb7/Twine/Twine2.download.recipe#L26

Looks like twinery.org now links to github.com instead of bitbucket.org for downloads: https://github.com/klembot/twinejs/releases/download/2.2.1/twine_2.2.1_osx.zip

**Comment by denmoff** [comment]
Thanks. I'll switch to using GitHubReleasesInfoProvider

**Comment by blackthroat** [comment]
Fixed and confirmed working with 621686213383d1eca159c36ab5ae41bb96d3841a

Thanks for the quick fix!
